osc/core.py

```diff
@@ -3748,7 +3752,15 @@ def copy_pac(
         if force_meta_update or not found:
             print('Sending meta data...')
             u = makeurl(dst_apiurl, ['source', dst_project, dst_package, '_meta'])
-            http_PUT(u, data=src_meta)
+            http_PUT(u, data=meta)
+
+    if meta is None:
+        meta = show_package_meta(dst_apiurl, dst_project, dst_package)
+
+    root = ET.fromstring(meta)
+    if root.find("scmsync") is not None:
+        print("Note: package source is managed via SCM")
```

Comment:
This code is broken. `show_package_meta()` returns a list and that doesn't work with `ET.fromstring(meta)`.
It should be: `root = ET.fromstring(b"".join(meta))`

Reply:
show_files_meta is maybe the more direct way, changed